spec/models/c100_application_spec.rb

```diff
@@ -28,6 +28,23 @@
     end
   end
 
+  describe '.online_submission?' do
```

Comment:
Duplicated tests. There is already a `describe '#online_submission?'`

Reply:
addressed